packages/gui/src/components/nfts/NFTContextualActions.tsx

```diff
@@ -558,6 +564,77 @@ function NFTBurnContextualAction(props: NFTBurnContextualActionProps) {
     </MenuItem>
   );
 }
+
+/* ========================================================================== */
+/*                     Invalidate cache of a single NFT                       */
+/* ========================================================================== */
+
+type NFTInvalidateContextualActionProps = NFTContextualActionProps;
+
+function NFTInvalidateContextualAction(
+  props: NFTInvalidateContextualActionProps,
+) {
+  const { onClose, selection } = props;
```

Comment:
`onClose` isn't passed in anymore.

```suggestion
  const { selection } = props;
```